examples/studio-kit-demo/src/host/host.tsx

```diff
@@ -210,6 +210,12 @@ const Project = () => {
     })
   }, [])
 
+  React.useEffect(() => {
+    return projectCommands.useLayerState('Background', (props) => {
```

Comment:
This method does not exist

Reply:
@hudbuddy Thanks for callout. Somehow it ended up in code. It's removed now